src/threads/table.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { dynamodb, toDBItem } from '../dynamodb';
+import { ForumTable } from './table';
+
+const putItem = (threadId: string, timestamp: number, data: Record<string, unknown>) => dynamodb.putItem({
+  TableName: 'forumTable',
```

Comment:
In particular, nothing guarantee in your tests that the db is empty before. I suppose you know technical that it is sure, but it would be better to cleanup before each test or at least to do the check.

Reply:
Fixed